### PR TITLE
PLANNER-2472 daily native build of optaplanner-quickstarts

### PR DESCRIPTION
**Pull request by Ginxo**
### JIRA

https://issues.redhat.com/browse/PLANNER-2472

### Referenced pull requests

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* for a <b>pull request build</b> please add comment: <b>Jenkins retest this</b>
* for a <b>full downstream build</b> please add the label `run_fdb`
* for a <b>specific pull request build</b> please add comment: <b>Jenkins (re)run [optaplanner-quickstarts] tests</b>
* for a <b>Quarkus LTS check</b> please add comment: <b>Jenkins run LTS</b>
* for a <b>specific Quarkus LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner-quickstarts] LTS</b>
* for a <b>Native check</b> please add comment: <b>Jenkins run native</b>
* for a <b>specific Native LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner-quickstarts] native</b>
</details>

### CODE REVIEW

File: .ci/jenkins/Jenkinsfile.native [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        cleanup {
+            script {
+                util.cleanNode('docker')
+            }
+        }
+    }
+}
+
+void sendNotification() {
+    mailer.sendMarkdownTestSummaryNotification("${NOTIFICATION_JOB_NAME}", "[${params.BUILD_BRANCH_NAME}] Optaplanner Quickstarts", [env.KOGITO_CI_EMAIL_TO])
```
Comment (rsynek) [inline .ci/jenkins/Jenkinsfile.native]: Let's also add the `optaplanner-team` to the cc (see https://github.com/kiegroup/optaplanner/blob/main/.ci/jenkins/Jenkinsfile.deploy#L296):
```suggestion
    mailer.sendMarkdownTestSummaryNotification("${NOTIFICATION_JOB_NAME}", "[${params.BUILD_BRANCH_NAME}] Optaplanner Quickstarts", [env.KOGITO_CI_EMAIL_TO], "cc @*optaplanner-team*")
```

**Review by rsynek — APPROVED** [review]

**Review by mareknovotny — APPROVED** [review]

**Comment by radtriste** [comment]
Was it really done for `stable` branch on purpose ?

**Comment by radtriste** [comment]
> Was it really done for `stable` branch on purpose ?

I mean `stable` branch is not handled yet by the DSL, only `development` or release branches. So we would need to add handling there

**Comment by Ginxo** [comment]
good catch @radtriste , here it is https://github.com/kiegroup/optaplanner-quickstarts/pull/257

**Pull request merged**